src/test/java/uk/gov/ons/ctp/common/event/EventPublisherTest.java

```diff
@@ -40,6 +44,8 @@
   private static final String ROUTING_KEY = "whereAreWeRoutingThis";
```

Comment:
ROUTING_KEY no longer used - remove

Reply:
Not part of the changes I made, was already present. Peter is making changes to the event publisher at the moment. As this branch is now merged he has made this change as part of his commit.